engine/crates/parser-sdl/src/connector_parsers.rs

```diff
@@ -82,13 +82,10 @@ pub(crate) fn merge_registry(ctx: &mut VisitorContext<'_>, mut src_registry: Reg
 
     let mut main_registry = ctx.registry.borrow_mut();
 
-    // I am sort of making the assumption that connectors won't generate names that
```

Comment:
fixed a while ago. they are checked if they're unique

Reply:
where is this done?

Reply:
Would be nicer to have this a bit more implicit. Now we have to add the iteration to that function for every new connector...

Reply:
That code doesn't do what this comment was referring to.  That code makes sure the names of connectors are unique - this comment is referring to the names of types generated by the connectors - I think this comment still stands.

Reply:
Good point. I'll return this comment.